com.github.kmkolasinski.AwesomeBump.json

```
@@ -0,0 +1,46 @@
+{
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "separate-locales": false,
+    "app-id": "com.github.kmkolasinski.AwesomeBump",
+    "command": "awesomebump",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--device=dri"
+    ],
+    "modules": [
+        {
+            "name": "AwesomeBump",
+            "no-autogen": true,
+            "no-make-install": false,
+            "no-parallel-make": false,
+            "sources": [
+                {                    
+                    "tag": "Linuxv5.1.1",
+                    "commit": "d6912bfcb5219844841c2582f799c9a6dddaf533",
+                    "type": "git",
+                    "url": "https://github.com/kmkolasinski/AwesomeBump"
+                },
+                {
+                    "commands": [
+                       "qmake ./AwesomeBump.pro", "make -j 8", 
```

```suggestion
                       "qmake ./AwesomeBump.pro", "make -j $FLATPAK_BUILDER_N_JOBS", 
```
Use $FLATPAK_BUILDER_N_JOBS instead of hardcoding the value here